dev/src/reference.ts

```diff
@@ -1458,6 +1458,33 @@ export class VectorQuerySnapshot<
   }
 }
 
+interface QueryStreamResponse<
+  AppModelType = firestore.DocumentData,
+  DbModelType extends firestore.DocumentData = firestore.DocumentData,
+> {
+  transaction?: Uint8Array;
+  readTime?: Timestamp;
```

Comment:
I don't think `readTime` ever exists outside the context of a snapshot. The snapshot should have a `readTime` property. So, we can remove from this object IMHO.

Reply:
This `QueryStreamResponse` represents the shape of any single "element" emitted from the stream. It is possible to the stream to emit only a `readTime` and no documents (empty result). So I think keeping `readTime` in this particular place is valid.

Reply:
👍 I stand corrected!